composer/workflows/integrated_example.py

```diff
@@ -0,0 +1,140 @@
+  # We need to create a BigQuery dataset to store the output from Dataflow.
+  # There's currently no BigQuery dataset creation operator to do this, so
+  # we use a `BashOperator` catch-all operator. Notice that the
+  # `create_bq_dataset_if_not_exist` command is implemented using the
+  # same command that we'd use in the BigQuery command line tools.
+  #
+  # After this completes, a BigQuery dataset with the `BQ_DATASET_NAME`
+  # should be viewable from your BigQuery dashboard.
+  create_bq_dataset_if_not_exist = """
+    bq ls {0}
+    if [ $? -ne 0 ]; then
+      bq mk {0}
+    fi
+  """.format(BQ_DATASET_NAME)
+  CREATE_BIGQUERY_DATASET = BashOperator(
+      task_id='create-bq-dataset',
+      bash_command=create_bq_dataset_if_not_exist,
+      dag=dag)
+
+  # This DataFlow operator runs a Python pipeline from Apache Beam,
+  # `hourly_team_score.py`, that calculates the scores of teams in batch and
+  # sends the results to our BigQuery dataset.
+  #
+  # Optional: We use this Apache pipeline file directly by setting the py_file as
+  # follows; however, if you'd like to see how the Dataflow oeprator uses local
+  # Python pipeline files, copy the pipeline file (https://github.com/apache/beam/blob/master/sdks/python/apache_beam/examples/complete/game/hourly_team_score.py)
+  # to a directory in your bucket (we use `pipelines/`):
+  #   gsutil hourly_team_score.py gs://{composer-bucket}/pipelines
+  # Then, modify the `py_file` argument in the given DAG to point to the local
+  # file location:
+  #   py_file=`/home/airflow/gcs/pipelines/hourly_team_score`
+  #
+  # Notice how we feed in the data file `game_data-condensed.csv` (located in a
+  # public bucket `gs://example-datasets) and point the pipeline to our BigQuery
+  # dataset. As this runs (should take 3-5 minutes), you should be able to track
+  # the progress in your Dataflow dashboard as a job with the prefix
+  # `run-dataflow-python`.
+  RUN_DATAFLOW_PYTHON = DataFlowPythonOperator(
+      task_id='run-dataflow-python',
+      py_file='apache_beam.examples.complete.game.hourly_team_score',
+      py_options=['-m'],
+      options={
+          'input': 'gs://example-datasets/game_data_condensed.csv',
+          'dataset': BQ_DATASET_NAME
+      }
+  )
+
+  # After the Dataflow process completes, you should see an `hourly-team-score`
+  # table within the `example-dataset`. We now want to filter the entries by
+  # running a query on this table using the BigQuery operator below. This query
+  # chooses all the teams with `total_score > 20000` from the table we just
+  # generated, `example_dataset.hourly_team_score`. We write the result of this
+  # query (`destination_dataset_table`) to a new table,
+  # `example_dataset.results`, and set `write_disposition="WRITE_TRUNCATE" (so
+  # our results will overwrite any results already existing in this table).
+  # Run a query on this dataset
+  RUN_BQ_QUERY = BigQueryOperator(
+      task_id='bq-team-query',
+      bql="""
```

Comment:
Did we backport the new `sql` keyword argument from Airflow 1.10? It'd be good to use that for future compatibility.